### ISSUE TITLE
I'm an amature building ruffle on Repl.it, however I keep getting this error

### ISSUE THREAD

**Issue by CBGamesdev** [ruffle-rs/ruffle#1832]
So, like in my last post, i'm building ruffle for my website (to clarify it's a google site).  However I switched over to Repl.it as I think it supports .wasm files. However, when trying to get my game loading, it replaces the flash player but when I click the play button, nothing happens. I'll put the error I got in the command box below:

"Serious error loading Ruffle: TypeError: Could not download wasm module
Serious error occurred loading SWF file: TypeError: Could not download wasm module" 

and

"Creating WebGL renderer...
Creating WebGL2 context.
WebGL graphics driver: Mesa DRI Intel(R) HD Graphics 500 (APL 2)
Minimum audio buffer sample rate: 5512
New Ruffle instance created.
Loading SWF file https://sites.google.com/site/chilibowlgames/bloxorz.swf"
Asynchronous error occurred: Could not fetch movie https://sites.google.com/site/chilibowlgames/bloxorz.swf%22"

Any help? And if you could, dumb it down. I'm very new to this.

**Comment by parkerlreed** [comment]
You have a " as part of the URL making it invalid.
